recipes/_compile.rb

```diff
@@ -78,6 +78,9 @@
   #   https://github.com/CentOS/sig-cloud-instance-images/issues/4
   #
   package 'tar'
+  # Don't introduce a bug on RHEL by not having bzip2
+  # this was fixed previously in omnibus v2.4.12
+  package 'bzip2'
```

Comment:
@yzl Technically one could say that I introduced this bug back again in 4.0.0 by pruning the code that ensured bzip2 was installed. I think removing the comment probably would be best in this case? I was being overly explanatory I think...